demo-node/entrypoint.bash

```diff
@@ -43,20 +43,16 @@ curl -X POST \
 #  -d "{\"name\": \"Live CVE Notifications from the NIST API\", \"definition\": \"${cve_pipe}\", \"start_when_created\": true}" \
 #  http://127.0.0.1:5160/api/v0/pipeline/create
 
-# The shell operator decompresses the data and writes it to `read suricata` on
-# the fly.
 # !! Not started because the data has already been imported while building the image.
-suricata_pipe="shell 'bash -c \\\"curl -s -L https://storage.googleapis.com/tenzir-datasets/M57/suricata.tar.zst | tar -x --zstd --to-stdout\\\"' | read suricata | where #schema != \\\"suricata.stats\\\" | import"
+suricata_pipe="tenzir 'load https https://storage.googleapis.com/tenzir-datasets/M57/suricata.json.zst | decompress zstd | read suricata | import'"
```

Comment:
This one doesn't contain the `where #schema != suricata.stats` that the one from `import.bash` has.

Reply:
should we not get rid of that filter now that we support larger events?

Reply:
I think we should keep it. The events are rather meaningless.